handler/src/test/java/io/netty/handler/ssl/SslContextBuilderTest.java

```diff
@@ -20,6 +20,7 @@
 import io.netty.util.CharsetUtil;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.function.Executable;
+import org.mockito.Mockito;
```

Comment:
```suggestion
```

We don't need to import mockito anymore

Reply:
Fixed!

[525fa19](https://github.com/netty/netty/pull/14058/commits/525fa19d9b0209564b530027921e26d832a163a1)